.github/workflows/ci.yml

```diff
@@ -337,8 +333,10 @@ jobs:
           - flaky
       fail-fast: false
     steps:
-      - name: Checkout project
+      - name: Checkout project with history
         uses: actions/checkout@v3
+        with:
+            fetch-depth: 0
```

Comment:
Is this required for `integration` test suite? I thought it only matters to the `build` or `compile`?

Reply:
Required because a shell script still need the git history to replace the rocks-native version tag in pom.xml in order to get the SNAPSHOT jar in Apache maven repo.